src/redux/pokedexSlice.js

```diff
@@ -0,0 +1,73 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+// Actions
+const GET_DATA = 'getPokemon';
+const GET_NEXT = 'getNext';
```

Comment:
**OPTIONAL** - In order to follow redux best practices, I suggest that you use the ducks pattern when naming your action i.e `const GET_DATA = redux/pokedexSlice/GET_DATA`. Please fix this 😄

Reply:
@mckent05 Thanks I changed this